iron-menu-behavior.html

```diff
@@ -90,16 +90,25 @@
     },
 
     /**
-     * Resets all tabindex attributes to the appropriate value based on the
+     * Reset attributes of the menu's new/changed collection of items:
+     *
+     * Reset all tabindex attributes to the appropriate value based on the
      * current selection state. The appropriate value is `0` (focusable) for
      * the default selected item, and `-1` (not keyboard focusable) for all
      * other items.
+     *
+     * Reset the aria-selected attribute based on the current selection state.
+     * As _applySelection notes, we need to set aria-selected to false for
+     * unselected items, even in their initial state, not just when they become
+     * deselected.
      */
-    _resetTabindices: function() {
+    _resetItemAttributes: function() {
```

Comment:
One underscore indicates protected for us (two indicates private). This technically constitutes a breaking API change. If we can verify it's not being overridden in elements we control that depend on this behavior, it's probably fine..